### ISSUE TITLE
[Android] back-button doesn't work as expected

### ISSUE THREAD

**Issue by blue-hope** [EddyVerbruggen/cordova-plugin-native-keyboard#89]
My environment: ionic 3

I expected that back-button in android environment works like popping the ionic navigation control stack or exiting the app. 
But when I add this plugin in my android ionic-app, and use keyboard as usual, back-button does not work.
Here's the situation where the problem occurs:

> _If user want to type something in app, he'll open the keyboard tap_

In this situation, back-button works.

> _When he types something on keyboard tap_

In this situation, back-button doesn't work.

> _If he enter the typed word, and the cursor in keyboard input box exists_

In this situation, back-button doesn't work.

> _At last, if he touches other side apart from keyboard and the cursor disappears_

In this situation, back-botton works

I think the back-button is not processed on android_layer or in-app layer, but handled within the plugin inside. 
 How can I solve this problem ?

**Comment by pcvengros** [comment]
same problem, behavior:
- before first use of keyboard all is working as it should = my handler on back button is working
- open keyboard first time in messenger -> result -> when you hit back button, application is exited immediately

**Comment by gregavola** [comment]
@pcvengros Did you ever figure out a resolution here?

**Comment by pcvengros** [comment]
No, still same issue with no resolution. Back button handler works correctly unless the nativekeyboard is closed/hiden. If keyboard is open it will stop working and on back button whole app closes.
